### ISSUE TITLE
A few Science Seminar page issues

### ISSUE THREAD

**Issue by jchate6** [LCOGT/main_website#8]
- [x] The "Back to Science Seminar home -->" link is dead in new (2019) pages. (the href is literally blank) The links on previously made (2018) pages still function. This link should return to the main seminar page. 

- [x] Can we please capitalize "Series" in the main header across the top of all of these pages so that it reads "Seminar Series"?

- [x] The address is broken at the city/state level and there is no space between CA and 93117. Could this be fixed to be a single line: "Goleta, CA 93117" please?

**Comment by Fingel** [comment]
Hi Joey,
You should talk to Jesus. He'll make your account able to access the admin page. We all pretty much fix typos when we find them, it's an easy CMS interface (like wordpress or similar).

**Comment by Fingel** [comment]
Well, one of the issues was a template issue.

The Back to Seminar Home link being broken is because the pages with the broken link were being created as top level pages, not under the Science Seminar parent page like the previous ones:

https://screenshots.firefox.com/dyq9SqWleHxqR3MA/lco.global

When creating science seminar pages in the future, make sure you create them under the Science -> Science Seminar parent pages.

I was able to fix the address issue by editing this page from the admin: 
https://lco.global/admin/mezzanine_blocks/richblock/21/change/
Did take a while to find that though.

Finally, the lowercase 's' in Science Seminar was hardcoded in the templates. I've resolved that in this PR:

https://github.com/LCOGT/main_website/pull/9
